lib/fine-types/app/Options/Check.hs

```diff
@@ -0,0 +1,44 @@
+{-# LANGUAGE ApplicativeDo #-}
+
+module Options.Check where
+
+import Prelude
+
+import Options.Applicative
+    ( InfoMod
+    , Parser
+    , header
+    , help
+    , long
+    , metavar
+    , progDesc
+    , short
+    , strOption
+    )
+import Options.Common (inputOption)
+
+data CheckOptions = CheckOptions
+    { optInput :: Maybe FilePath
+    , optDir :: FilePath
+    }
+
+checkDescr :: InfoMod a
+checkDescr =
+    mconcat
+        [ progDesc "Check a fine-types module for errors"
+        , header "fine-types check - check a fine-types module for errors"
+        ]
+
+checkOptions :: Parser CheckOptions
+checkOptions = CheckOptions <$> inputOption <*> dirOption
+
+dirOption :: Parser FilePath
+dirOption =
+    strOption
+        ( mconcat
+            [ long "dir"
+            , short 'd'
+            , metavar "DIR"
```

Comment:
```suggestion
            , metavar "DIR"
            , value "."
```

This would be a sensible default value. 🤔

Reply:
ok

Reply:
```
paolino@plutimus:~/iohk/fine-types/lib/fine-types/test/data/Cardano/Ledger$ fine-types check -i ./Shelley.fine
check: Checking...
check: Success!
```